Mgetz10/cdc-open-viz
# Feat/mobile height

## Testing Steps
Method 1
1. Open Storybook
2. enter small mobile view mode
3. add 'mobileVertical' To a chart component
4. give value and notice change

Method 2
1. Open editor
2. Continue through chart creation until configure section
3. Adjust the "Mobile Height" field under "General"
4. Adjust window width so that the preview viewport width is <=350 to see changes

## Self Review
- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)
<img width="372" alt="Screenshot 2024-08-20 at 12 20 31 PM" src="https://github.com/user-attachments/assets/25689633-f922-4c3c-9863-6461862b8556">
<img width="807" alt="Screenshot 2024-08-20 at 11 04 47 AM" src="https://github.com/user-attachments/assets/6fbc8d79-fbff-4c89-bbfb-1c04bd173402">
<img width="810" alt="Screenshot 2024-08-20 at 11 03 59 AM" src="https://github.com/user-attachments/assets/c763511e-9aa9-423b-a3c8-eede8a34a8f4">


## Additional Notes

If "Mobile Height" is not set, will default to "vertical"
